sagemaker-fundamentals/create-training-job/create_training_job_encrypt_static_assets.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>Should be executable cell instead of markdown?</p><p> </p><p>When I try to execute it, I get the error message:</p><pre class="ql-syntax" spellcheck="false">Traceback (most recent call last):
  File "/usr/bin/train", line 21, in &lt;module&gt;
    s3 = sess.client('s3')
  File "/opt/conda/lib/python3.8/site-packages/boto3/session.py", line 258, in client
    return self._session.create_client(
  File "/opt/conda/lib/python3.8/site-packages/botocore/session.py", line 797, in create_client
    region_name = self._resolve_region_name(region_name, config)
  File "/opt/conda/lib/python3.8/site-packages/botocore/session.py", line 853, in _resolve_region_name
    validate_region_name(region_name)
  File "/opt/conda/lib/python3.8/site-packages/botocore/utils.py", line 1016, in validate_region_name
    raise InvalidRegionError(region_name=region_name)
botocore.exceptions.InvalidRegionError: Provided region_name '$region_name' doesn't match a supported format.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>